FlorenceBML/BMLacq811.xml

```diff
@@ -30,7 +30,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     <msIdentifier>
                         <repository ref="INS0339BML"/>
                         <collection>Acquisizioni e doni</collection>
-                        <idno>BML Acq. e doni 811</idno>
+                        <idno facs="Laurenziana/BML_007/BML_007-" n="225">BML Acq. e doni 811</idno>
```

Comment:
please remove facs, there are no images for 811, BML_007 are images for 812

Reply:
OK.